docs/spec_changelog.md

```diff
@@ -0,0 +1,43 @@
+---
+id: "spec_changelog"
+title: "Twirp Wire Protocol Changelog"
+sidebar_label: "Changelog"
+---
+
+This document lists changes to the Twirp specification.
+
+## Changed in v6
+
+### URL scheme
+
+In [v5](./PROTOCOL.md), URLs followed this format:
+
+```bnf
+**URL ::= Base-URL "/twirp/" [ Package "." ] Service "/" Method**
+```
+
+Version 6 changes this format to remove the mandatory `"/twirp"` prefix, and
+changes `Package` and `Service` to be delimited with a `/` instead of a `.`
```

Comment:
Packages will still contain `.` characters; we're not changing the internal delimiter. How about this phrasing:

"and changes the separator between `Package` and `Service` to be a `/` instead of a `.` character"